root/static/scripts/common/utility/parseIntegerOrNull.js

```diff
@@ -9,7 +9,7 @@
 
 import parseInteger from './parseInteger';
 
-export default function parseIntegerOrNull(str: string): $FlowFixMe | null {
+export default function parseIntegerOrNull(str: string): number | null {
```

Comment:
This was just an outright bug. Concerning since this module is marked as strict.